components/form/builder/mocks/index.js

```diff
@@ -0,0 +1,108 @@
+export const FORM_BUILDER_SELECT_FIELD_MOCK = {
```

Comment:
Maybe this file could fit better in `demo` dir instead `src`

Reply:
Hi @javiauso, thanks for your reply 🙏🏻 

This mock files are not inside `src` file. This is inside new `mocks` folder and ignored from `npm` file to avoid increase npm package. The main reason of that is to share mocks between `demo` and `tests` contexts instead of have it inside `demo` folder. 

That's why I create this new contextual folder.

Reply:
Ups you're right!! 😆